pagerduty/resource_pagerduty_user_contact_method.go

```diff
@@ -30,6 +30,7 @@ func resourcePagerDutyUserContactMethod() *schema.Resource {
 				ValidateFunc: validateValueFunc([]string{
 					"email_contact_method",
 					"phone_contact_method",
+					"push_notification_contact_method",
```

Comment:
does this also need to be added to the docs?

Reply:
Documentation should be up-to-date now :)